specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/Administrators.json

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
Please address all the lintdiff errors and remove the ARMChangesRequested label from the PR once done.

Reply:
[https://github.com/Azure/azure-rest-api-specs/pull/20212/checks?check_run_id=8280219057](https://github.com/Azure/azure-rest-api-specs/pull/20212/checks?check_run_id=8280219057)

Reply:
@rkmanda do you want us to fix all staging linterrors before we proceed with this PR? Staging linterrors shouldn't be a blocker to proceed right?